jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBQueryResultSet.java

```diff
@@ -716,10 +689,75 @@ public boolean next() throws SQLException {
     if (emptyResultSet) {
       return false;
     }
-    record = Utils.getRowRecord(byteBuffer, columnTypeDeduplicatedList);
+    constructOneRow();
     return true;
   }
 
+  private void constructOneRow() {
+    record[0] = new byte[Long.BYTES];
```

Comment:
for the timestamp column, the record[0] could be created by a new byte[Long.BYTES] and reused. No need to new a byte[Long.BYTES] each time.